packages/framework/src/DataFixtures/Demo/ProductDataFixtureLoader.php

```diff
@@ -383,4 +383,17 @@ private function addZeroPricesForPricingGroupsThatAreMissingInDemoData($demoData
 
         return $demoDataManualPrices;
     }
+
+    /**
+     * @param int $domainId
+     * @return int
+     */
+    private function getCsvProductColumnNameByDomainId(int $domainId)
```

Comment:
I believe it should work similarly to `getDescriptionColumnForDomain()` - so when there is another locale than `cs` or `en`, it should throw `\Shopsys\FrameworkBundle\Component\DataFixture\Exception\UnsupportedLocaleException`